docs/releasenotes/8.2.0.rst

```diff
@@ -1,6 +1,60 @@
 8.2.0
 -----
 
+Security
+========
+
+These issues were all found with `OSS-Fuzz`_.
+
+:cve:`2021-25287`, :cve:`2021-25288`, :cve:`2021-28675`: OOB read in Jpeg2KDecode
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* For J2k images with multiple bands, it's legal to have different widths for each band,
+  e.g. 1 byte for ``L``, 4 bytes for ``A``.
+* This dates to Pillow 2.4.0.
+
+:cve:`2021-28675`: DOS attack in PsdImagePlugin
```

Comment:
General comment: we've lost "Fix" for CVEs, but not for the non-CVE below. Intentional?

Reply:
Definitely not intentional